CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Ways Of Working
+
+Transport for the North (TfN) encourages a collaborative way of working and as such is happy
+to accept contributions on various aspects of our repositories, including:
+
+- Code maintenance and development
+- Feature requests and bug reporting
+- Documentation feedback and suggestions
+- Code examples and use cases
+
+## Feedback & Requests
+
+If you have any feedback or requests on the code, documentation, or anything else, please create a GitHub
+issue. We have created some issue templates to guide you through the information needed for different types of suggestions, and opens the discussions for collaboration.
+
+## Version Control
+
+CAF codebases are all hosted on [GitHub](https://github.com/Transport-for-the-North)
+and use [Git](https://git-scm.com/) for version control, see CAF Handbook's
+[GitHub section](https://transport-for-the-north.github.io/CAF-Handbook/contribution/github_wow/overview.html)
+for guidance.
```

Comment:
```suggestion
for guidance.

> [!NOTE]
> Code [releases](#releases) are managed using Semantic Versioning.
```

I'm aware that the word versioning can be ambiguous here, so trying to make it clearer for those less in the know!